geotrek/common/parsers.py

```diff
@@ -461,20 +461,22 @@ def parse(self, filename=None, limit=None):
                 self.add_warning(str(e))
         self.end()
 
-    def get_or_retry(self, url, params=None, authent=None):
-        try_get = self.number_of_try
+    def request_or_retry(self, url, verb='get', params=None, authent=None):
+        try_get = settings.PARSER_NUMBER_OF_TRIES
         assert try_get > 0
         while try_get:
-            response = requests.get(url, params=params, auth=authent)
-            if response.status_code == 503:
-                sleep(self.sleep_time)
+            action = getattr(requests, verb, None)
```

Comment:
Le dernier paramètre me semble inutile car si on utilise None ici on va de toute façon planter plus loin. Sauf que le message sera beaucoup moins clair : "None is not callable" au lieu de "No attribut foo on requests"